Cargo.lock

```diff
@@ -429,6 +447,19 @@ dependencies = [
  "serde_json",
 ]
 
+[[package]]
+name = "cargo_metadata"
```

Comment:
We'll probably have to do a coordinated dep update for cargo_metadata and compiletest_rs.

Reply:
Old version for cargo_metadata (with bunch of other stuff) can be dropped when rustfmt will be updated in rust repo (dependencies already updated in rustfmt repo).

Reply:
cargo_metadata in rustfmt seems to still use version 0.12. instead of 0.14. Is this intentional?

With `rustfmt` updated, we'd also need to update the tidy script and RLS.

Reply:
It was updated to 0.12 as it was most common version in workspace and required less changes, in case if it will updated to newer version.